ts/src/hyperliquid.ts

```diff
@@ -924,16 +951,19 @@ export default class hyperliquid extends Exchange {
         //         "oraclePx": "2367.3",
         //         "premium": "0.00090821",
         //         "prevDayPx": "2381.5"
+        //         "collateralToken": "0" hip3 tokens only
```

Comment:
collateralToken also exists for regular non-hip3 HL's perps. But it's always 0 (USDC) anyways